README.md

```diff
@@ -110,6 +118,47 @@ If everything has worked so far, you should now be able to create your own byted
 4. Now you should be in your own bytedeck site!
 5. If you would like to stop the project, use `Ctrl + C` in the command lines, then wait for each of the containers to stop.
 
+#### Alternative Method 1: Web Form
+Alternatively, if the above doesn't work for you, you can create a new tenant via the web form:
+
+0. If the server isn't already running, run it with: `python src/manage.py runserver` or `docker compose up web` (and ignore the link it tells you to access the page)
+1. Go to create a new deck at http://localhost:8000/decks/new/.
+2. Fill out all the empty boxes with your info and hit the 'create' button at the bottom.
+3. You should now be at the login, you can log in using log in using
+   - username: admin
+   - password: password (this is defined in TENANT_DEFAULT_SUPERUSER_PASSWORD in the .env file)
+4. Now you should be in your own bytedeck site!
+5. If you would like to stop the project, use `Ctrl + C` in the command lines, then wait for each of the containers to stop.
+
+#### Alternative Method 2: Using Django Management Command
```

Comment:
I don't think this 2nd option is necessary.  Goal of this readme is the minimum to get a new developer up and running, so more detail the main method is good.  No need for alternates.